### PR TITLE
Fix/specs

### PR DESCRIPTION
**Pull request by hugobast**
This is WIP

One of the goals I want to reach for this effort is to clean it up well and good and have working specs so that I can move this code back into Glossier.

- [x] Lint everything (except metrics)
- [x] Green up the specs (local and Travis)
- [ ] Check the test coverage and improve the coverage
- [ ] Stretch goal try and see what we can remove that Glossier isn't using

I'm hoping this would be useful: https://github.com/joshuaclayton/unused

### CODE REVIEW

File: lib/hushed/black_tie_bundle.rb [hunk 1]
```diff
@@ -2,32 +2,30 @@
 
 module Hushed
   module BlackTieBundle
```
Comment (aamyot) [inline lib/hushed/black_tie_bundle.rb]: we can just delete this... Black Tie set was a promotions for the Holidays

Comment (hugobast) [inline lib/hushed/black_tie_bundle.rb]: Glad you bring this up; I'll make a PR that removes it.

File: lib/hushed/black_tie_bundle.rb [hunk 2]
```diff
@@ -2,32 +2,30 @@
...
-    extend self
+    module_function
```
Comment (aamyot) [inline lib/hushed/black_tie_bundle.rb]: any reason why you prefer module_function over self?

File: spec/unit/black_tie_bundle_spec.rb [hunk 3]
```diff
@@ -3,16 +3,15 @@
 
 module Hushed
   describe Hushed::BlackTieBundle do
```
Comment (aamyot) [inline spec/unit/black_tie_bundle_spec.rb]: same here... we should delete

File: lib/hushed/black_tie_bundle.rb (hunk 2 above)
Comment (hugobast) [inline lib/hushed/black_tie_bundle.rb]: There's a commit that provides wider context on it 0463799 it's squashed so it'a a bit buried but the starting point is that Rubocop favors it.

File: spec/unit/black_tie_bundle_spec.rb (hunk 3 above)
Comment (hugobast) [inline spec/unit/black_tie_bundle_spec.rb]: I'll do a subsequent PR for the BackTieBundle removal.

**Pull request merged**